ui/static/ui/js/manage_taxonomies.jsx

```diff
@@ -50,7 +59,7 @@ define('manage_taxonomies', ['react', 'lodash', 'jquery', 'utils', 'bootstrap'],
      * If user selects edit button then open edit mode
      * Else call api to update term.
      */
-    formatHandler: function() {
+    editTerm: function() {
       var formatActionState = this.state.formatActionState;
 
       if (formatActionState === 'show') {
```

Comment:
It looks like this callback is only activated if `formatActionState === 'show'` so you can remove this line and the closing bracket.